### DISCUSSION TITLE
Correct texture workflow

### DISCUSSION THREAD

**Discussion by ralphmeijerambassadors** [Autodesk/maya-usd#3199] [Q&A]
Hi,

We are building our USD pipeline and running into some issues/questions. Currently we are trying to export assets from Houdini using usdPreviewSurface for animators to see a shaded environment while animating. However we are wondering what's the correct workflow to use .exr files for textures. Currently when we export the shading with exrs, viewport 2.0 will draw the shaders with the fallback color on the usdUvTexture. When we set up the textures with jpgs, things in the viewport will show correctly. 

I'm wondering if an exr workflow is supported for viewport 2.0 or if we should stick to jpgs for now. If it's supported what's the correct workflow to make them show up in vp2.0. We are using maya 2023.3 with MayaUSD-0.23

Thanks in advance.

Ralph

(left is exr with fallback color to red, right jpg)
![Screenshot from 2023-07-05 23-03-26](https://github.com/Autodesk/maya-usd/assets/94980933/f7a77fb1-cb00-4332-8e90-3da49e362fa8)

**Comment by sharktacos** [comment]
What type of file node are you using for these? USDuvTexture or aiImage ?

**Reply by ralphmeijerambassadors** [reply]
We are using UsdUvTexture currently just to be flexible to other DCC as well. We also tried materialX but preview isn't also not really working.

**Comment by wallworm** [comment]
Thanks for sharing this with us. I'll bring it up with the team.
